remodel/decorators.py

```diff
@@ -1,4 +1,4 @@
-class cached_property(object):
+class CachedProperty(object):
```

Comment:
I just went with pycharm suggestion on this one.  "Class names should use camel case convention."  But understand decorators when applied generally lower case. Reverted.